lib/src/summary.dart

```diff
@@ -126,6 +127,15 @@ class Summary {
 
     return map;
   }
+
+  PlatformSummary getPlatformSummary() {
+    Map<String, Platform> libraries = {};
+    dartFiles.values.forEach((dfs) {
+      libraries[dfs.uri] = dfs.platform;
```

Comment:
Should we filter to be only non-src files?

I'm crashing because I'm hitting src files from analysis that have a `null` `platform` value.